lib/linguist/languages.yml

```diff
@@ -3488,6 +3488,15 @@ Kickstart:
   - ".ks"
   tm_scope: source.kickstart
   language_id: 692635484
+Kind:
+  type: programming
+  color: "#329866"
+  extensions:
+  - ".kind"
+  - ".kind2"
```

Comment:
You've not included any samples for this extension.

Reply:
This extension is used by the older versions of the language, we are going to change all of the `.kind2` files to `.kind` so it's just a anticipation. Can we just add an PR in the future if we change `.kind2` to `.kind` definitely or we can maintain that antecipation?

Reply:
> we are going to change all of the `.kind2` files to `.kind` so it's just a anticipation. 

You'll only be able to change your files. Any other files owned by other users will still remain until they change the extension 😁.

> Can we just add an PR in the future if we change `.kind2` to `.kind` definitely or we can maintain that antecipation?

Once support for an extension is added, it remains. We never remove or change extensions as this will remove recognition for files previously recognised.

So you can start off with adding just the most used extension now (assuming it meets our popularity requirements) and submit a new PR later when the other extension has met our usage requirements.

On that topic, I've taken a quick look at the popularity, and I don't think either are popular enough for inclusion right now which means we won't merge this PR. The `HigherOrderCO` user is disproportionately influencing the numbers as they seem to own most of the files currently indexed.

Reply:
> On that topic, I've taken a quick look at the popularity, and I don't think either are popular enough for inclusion right now which means we won't merge this PR. The HigherOrderCO user is disproportionately influencing the numbers as they seem to own most of the files currently indexed.
Yeah, HigherOrderCO owns most part of the ecosystem of the language as it is widely used for internal projects, so I guess that we can wait a little bit more time in order to add the language to this repo :)